tests/pipeline/test_blaze.py

```diff
@@ -778,6 +776,44 @@ def _test_id(self, df, dshape, expected, finder, add):
             check_dtype=False,
         )
 
+    def _test_id_macro(self, df, dshape, expected, finder, add):
+        dates = self.dates
+        expr = bz.data(df, name='expr', dshape=dshape)
+        loader = BlazeLoader()
+        ds = from_blaze(
+            expr,
+            loader=loader,
+            no_deltas_rule='ignore',
+            missing_values=self.missing_values,
+        )
+
+        p = Pipeline()
+        macro_inputs = []
+        for column_name in add:
```

Comment:
Regarding comment below on ordering; maybe just scrap `add` and do `for column_name in expected.columns` instead?